sql/core/src/test/resources/sql-tests/results/pgSQL/date.sql.out

```diff
@@ -198,17 +198,29 @@ struct<DATE '1999-01-18':date>
 -- !query 21
 SELECT date '1999 Jan 08'
 -- !query 21 schema
-struct<DATE '1999-01-01':date>
+struct<>
 -- !query 21 output
-1999-01-01
```

Comment:
The result is wrong, it is better to throw the exception because the format with spaces (`yyyy [m]m [d]d`) is not supported currently.

Reply:
Oops. I missed this. Thank you for the catching this. 
cc @wangyum , too.